### ISSUE TITLE
Demo doesn't work locally

### ISSUE THREAD

**Issue by iyromanov** [godka/kurento-rtmp#14]
Hello!
Thank you for your work.
I've installed ffmpeg and ran KMS in Docker, but this demo doesn't work locally.

This is fragment of console output with problem:
```
Senging message: {"id":"onIceCandidate","candidate":{"candidate":"candidate:1544065162 1 tcp 1518280447 10.202.128.84 9 typ host tcptype active generation 0 ufrag GZSJ network-id 1 network-cost 10","sdpMid":"audio","sdpMLineIndex":0}}
From ffmpeg:
'[sdp @ 0x7ff15f000000] Could not find codec parameters for stream 1 (Video: h264, none): unspecified size
Consider increasing the value for the 'analyzeduration' and 'probesize' options
'
From ffmpeg:
'Guessed Channel Layout for Input Stream #0.0 : mono
'
From ffmpeg:
'Input #0, sdp, from '/Users/igor/Projects/kurento-rtmp/127.0.0.1_55012.sdp':
  Metadata:
    title           : KMS
  Duration: N/A, bitrate: 64 kb/s
    Stream #0:0: Audio: pcm_mulaw, 8000 Hz, mono, s16, 64 kb/s
    Stream #0:1: Video: h264, none, '
From ffmpeg:
'90k tbr, 90k tbn, 180k tbc
'
From ffmpeg:
'Output #0, flv, to 'rtmp://localhost/live/127.0.0.1_55012':
  Metadata:
    title           : KMS
    encoder         : Lavf58.22.100
'
From ffmpeg:
'    Stream #0:0: Audio: pcm_mulaw ([8][0][0][0] / 0x0008), 8000 Hz, mono, s16, 64 kb/s
Stream mapping:
  Stream #0:0 -> #0:0 (copy)
Press [q] to stop, [?] for help
'
From ffmpeg:
'/Users/igor/Projects/kurento-rtmp/127.0.0.1_55012.sdp: Operation timed out
```

Do you have any ideas why?

**Comment by godka** [comment]
Thx for your interest.
Please check the camera alive. This demo will not work on the laptop without the camera.

**Comment by iyromanov** [comment]
@godka camera is alive in my macbook pro. And the same effect in my android phone

**Comment by NiKlimenko** [comment]
@godka the same issue is reproduced. Btw, the demo doesn't work also. Could you review the whole demo?
To fix the client part need to replace `URL.createObjectURL` with just `event.stream`
```js
videoElement.srcObject = event.stream
```

**Comment by godka** [comment]
> @godka the same issue is reproduced. Btw, the demo doesn't work also. Could you review the whole demo?
> To fix the client part need to replace `URL.createObjectURL` with just `event.stream`
> 
> ```js
> videoElement.srcObject = event.stream
> ```

Thx. Please pull a request for this issue~

**Comment by NiKlimenko** [comment]
For those who are faced with the problem that @iyromanov described above try to install and run KMS directly in your OS instead of using the docker image.

**Comment by NiKlimenko** [comment]
@godka I've pulled a request to fix demo #15

**Comment by godka** [comment]
@NiKlimenko Thx
